### PR TITLE
Check-in FAQ

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: docs/en/faq.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+1. You haven't installed lmdeploy's precompiled package. `_turbomind` is the pybind package of c++ turbomind, which involves compilation. It is recommended that you install the precompiled one.
+
+```shell
+pip install lmdeploy
+```
+
+2. If you have installed it and still encounter this issue, it is probably because you are executing turbomind-related command in the root directory of lmdeploy source code. Switching to another directory will fix it
+
+## Turbomind Inference
```
Comment (AllentDan) [inline docs/en/faq.md:25]: shall we add `libnccl.so.2` not found FAQ? I also met this yesterday with lmdeploy 0.0.5.

**Comment by irexyc** [comment]
ImportError: /home/PJLAB/maningsheng/anaconda3/lib/python3.8/site-packages/lmdeploy/lib/_[turbomind.cpython-38-x86_64-linux-gnu.so](http://turbomind.cpython-38-x86_64-linux-gnu.so/): symbol cudaFreeAsync version libcudart.so.11.0 not defined in file libcudart.so.11.0 with link time reference

minimal cuda version 11.2

**Review by irexyc — APPROVED** [review]

**Comment by lvhan028** [comment]
> ImportError: /home/PJLAB/maningsheng/anaconda3/lib/python3.8/site-packages/lmdeploy/lib/_[turbomind.cpython-38-x86_64-linux-gnu.so](http://turbomind.cpython-38-x86_64-linux-gnu.so/): symbol cudaFreeAsync version libcudart.so.11.0 not defined in file libcudart.so.11.0 with link time reference
> 
> minimal cuda version 11.2

updated

**Review by AllentDan — APPROVED** [review]
LGTM

**Pull request merged**